packages/documentation/src/stories/components/table/table.stories.tsx

```diff
@@ -9,10 +10,181 @@ export default {
       page: docsPage,
     },
   },
-  args: {},
-  argTypes: {},
+  args: {
+    variant: 'null',
+    caption: 'List of users',
+    captionPlacement: 'bottom',
+    alignment: 'align-top',
+    hover: false,
+  },
+  argTypes: {
+    caption: {
+      name: 'Caption',
+      description: 'A description of the table, very useful for screen reader users.',
+      control: {
+        type: 'text',
+      },
+      table: {
+        category: 'Caption',
+      },
+    },
+    captionPlacement: {
+      name: 'Caption Placement',
+      description:
+        'Choose where to place the description of the table. If a visual description is not necessary, it can also be hidden visually and still be of value for screen reader users.',
+      control: {
+        type: 'select',
+        labels: {
+          bottom: 'Bottom (default)',
+          top: 'Top',
+          hidden: 'Visually hidden',
+        },
+      },
+      options: ['bottom', 'top', 'hidden'],
+      table: {
+        category: 'Caption',
+      },
+    },
+    variant: {
+      name: 'Variants',
+      description: 'Stylistic table variants.',
+      control: {
+        type: 'radio',
+        labels: {
+          'null': 'Default',
+          'table-striped': 'Striped',
+          'table-bordered': 'Bordered',
+          'table-borderless': 'Borderless',
+        },
+      },
+      options: ['null', 'table-striped', 'table-bordered', 'table-borderless'],
+      table: {
+        category: 'General',
+      },
+    },
+    alignment: {
+      name: 'Alignment',
+      description:
+        'Vertical alignment options for cell content. Alignment classes can also be set for individual rows or cells.',
+      control: {
+        type: 'select',
+        labels: {
+          'align-top': 'Top (default)',
+          'align-middle': 'Middle',
+          'align-bottom': 'Bottom',
+        },
+      },
+      options: ['align-top', 'align-middle', 'align-bottom'],
+      table: {
+        category: 'General',
+      },
+    },
```

Comment:
fixed the defaults for this variant

Reply:
The effect of the `align-middle` class is still not visually evident, on the default story, because there is now row, that contains multiline content. But at least it is recoginzable now in the code.